frontend/src/components/bars/Bar/index.tsx

```diff
@@ -20,7 +30,15 @@ const Bar = (): ReactElement => {
 
     return (
         <Box className={classes.root} boxShadow={2}>
+            <Box>
```

Comment:
Useful box ?

Reply:
Yes, because its parent has a CSS selector that targets it. In the top bar there are 3 flex-evenly-spaced divs. This particular one will contain the buttons on the left-hand side of the top bar : 

![top_bar](https://user-images.githubusercontent.com/45355989/107614604-eb89fa00-6c4a-11eb-87cf-daecfc115bc0.png)